examples/cis/main.tf

```diff
@@ -0,0 +1,94 @@
+locals {
+  email          = "terraform-ci@suse.com"
+  identifier     = var.identifier
+  name           = "tf-rke2-${local.identifier}"
+  username       = "tf-${local.identifier}"
+  file_path      = "tf-${local.identifier}"
+  extra_config   = file("${abspath(path.root)}/config.yaml")
+  prep_script    = file("${abspath(path.root)}/prep.sh")
+  rke2_version   = var.rke2_version # I want ci to be able to get the latest version of rke2 to test
+  public_ssh_key = var.key          # I don't normally recommend using variables in root modules, but it allows tests to supply their own key
+  key_name       = var.key_name     # A lot of troubleshooting during critical times can be saved by hard coding variables in root modules
+  # root modules should be secured properly (including the state), and should represent your running infrastructure
+}
+resource "random_uuid" "join_token" {}
+# selecting the vpc, subnet, and ssh key pair, generating a security group specific to the ci runner
+module "aws_access" {
+  source              = "rancher/access/aws"
+  version             = "v0.0.8"
+  owner               = local.email
+  vpc_name            = "default"
+  subnet_name         = "default"
+  security_group_name = local.name
+  security_group_type = "egress" # https://github.com/rancher/terraform-aws-access/blob/main/modules/security_group/types.tf
+  ssh_key_name        = local.key_name
+}
+
+module "aws_server" {
+  depends_on          = [module.aws_access]
+  source              = "rancher/server/aws"
+  version             = "v0.0.16"
+  image               = "rhel-8-cis" # https://github.com/rancher/terraform-aws-server/blob/main/modules/image/types.tf
+  owner               = local.email
+  name                = local.name
+  type                = "small" # https://github.com/rancher/terraform-aws-server/blob/main/modules/server/types.tf
+  user                = local.username
+  ssh_key             = local.public_ssh_key
+  ssh_key_name        = local.key_name
+  subnet_name         = "default"
+  security_group_name = module.aws_access.security_group_name
+}
+
+module "config" {
+  source            = "rancher/rke2-config/local"
+  version           = "v0.0.5"
```

Comment:
bump to "v0.0.8" ?

Reply:
the config module is only on v0.0.5: https://registry.terraform.io/modules/rancher/rke2-config/local/latest
https://github.com/rancher/terraform-local-rke2-config